about.html

```diff
@@ -58,16 +86,22 @@
 
       <div class="btn-box">
         <button class="switch-mode">
-          <img src="./img/btn-icons/switch.png" alt="" />
+          <svg class="icon switch-ico">
+            <use xlink:href="sprite.svg#switch"></use>
+          </svg>
         </button>
 
-        <a href="https://discord.com/" class="header-link" target="_blank">
-          <img src="./img/socialntw-icons/discord-btn.png" alt="" />
+        <a href="#" class="basic-btn media-btn">
+          <svg class="icon discord-btn">
+            <use xlink:href="sprite.svg#discord-btn"></use>
+          </svg>
           DISCORD
         </a>
 
-        <button class="header-btn">
-          <img src="./img/btn-icons/wallet.png" alt="" />
+        <button class="basic-btn btn-light">
```

Comment:
One more point about classes: you should choose the approach to name the buttons in this case - `btn` at the beginning or at the end, do not mix. For example - `btn-basic btn-light` or `basic-btn light-btn`

Reply:
Thank you for comment about consistency in the names!

#fixed